Ladybird/BrowserWindow.cpp

```diff
@@ -333,7 +333,7 @@ void BrowserWindow::debug_request(DeprecatedString const& request, DeprecatedStr
     m_current_tab->debug_request(request, argument);
 }
 
-void BrowserWindow::new_tab(QString const& url, Activate activate)
+Tab* BrowserWindow::new_tab(QString const& url, Activate activate)
```

Comment:
This is never null and should be a reference

Reply:
yes, fixed.